src/Form/matching.js

```diff
@@ -717,17 +705,16 @@ const getRelatedText = (el, form, cssSelector) => {
     if (container === el || container.nodeName === 'SELECT') return ''
 
     // If the container has a select element, remove its contents to avoid noise
-    const noisyText = container.querySelector('select')?.textContent || ''
-    const sanitizedText = removeExcessWhitespace(container.textContent?.replace(noisyText, ''))
```

Comment:
I think `extractElementStrings` doesn't use `removeExcessWhitespace`, which does more than just `trim`. Can you look into that, please?

Reply:
I've added `removeExcessWhitespace` too, things like this are fast enough